cypress/integration/login.js

```diff
@@ -1,26 +1,30 @@
-describe("Login functionality", function() {
-  it("Should display validation for empty user after attempted loggin", function() {
-    cy.visit("/");
-    cy.get(".Select.not-valid").should("not.visible");
-    cy.get('[type="submit"]').click();
-    cy.get(".Select.not-valid").should("be.visible");
-  });
+const date = new Date();
+const today = date.getDate();
 
-  it("Should be able to login with role User", function() {
-    cy.get('[id="loginForm.userId"]').click({ force: true });
-    cy.get('[aria-label="Demo User"]').click();
-    cy.get('[id="loginForm.role"]').click({ force: true });
-    cy.get('[aria-label="User"]').click();
-    cy.get('[type="submit"]').click();
+describe('Login functionality', function() {
+    it('Should display validation for empty user after attempted loggin', function () {
+       
+        cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click()
+        cy.get('.Select.not-valid').should('be.visible')
+    })
 
-    cy.url().should("include", "/time-logging");
-    cy.get(".page__title").contains("Timesheets");
-    cy.get(".calendar").should("be.visible");
-    cy.get(".tile.form").should("be.visible");
-    cy.get(".user-info__title").contains("Demo User");
-    cy.get(".main-nav")
-      .find("li")
-      .should("have.length", 1);
-    cy.get(".calendar__date").should("have.css", "background-color");
-  });
-});
+    it('Should be able to login with role User', function () {
+        cy.get('[id="loginForm.userId"]').click({force:true})
+        cy.get('[aria-label="Demo User"]').click()
+        cy.get('[id="loginForm.role"]').click({force:true})
+        cy.get('[aria-label="User"]').click()
+        cy.get('[type="submit"]').click()
+
+        cy.url().should('include', '/time-logging')
+        cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar').should('be.visible')
+        cy.get('.tile.form').should('be.visible')
+        cy.get('.user-info__title').contains('Demo User')
+        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.calendar__date').should('have.css', 'background-color')
+        cy.get('[data-test="timesheets-current-day"]').contains(today)
```

Comment:
[data-test="timesheets-current-day"] - is this locator unique for today?

Reply:
I have changed to .calendar__date

Reply:
Done